test/integration/auth/auth.prose.test.ts

```diff
@@ -376,7 +376,8 @@ describe('Authentication Spec Prose Tests', function () {
           expect(stats).to.exist;
         });
 
-        it(
+        // TODO(NODE-6752): Fix flaky test
+        it.skip(
```

Comment:
At the bottom of the test can you add `.skipReason = "..."` so the TODO shows up in logs too